rootfs-builder/rootfs.sh

```diff
@@ -159,7 +159,9 @@ done
 
 shift $(($OPTIND - 1))
 
+# Fetch the first element from GOPATH
 [ -z "$GOPATH" ] && die "GOPATH not set"
+GOPATH_LOCAL="${GOPATH%%:*}"
```

Comment:
I know this is documented in the commit, but please could you add a comment here explaining why only the first path element is being used.

Reply:
Ok. I will add relevant explanation😊